src/backend/distributed/sql/udfs/get_cluster_clock/latest.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TABLE pg_catalog.pg_dist_local_group ADD COLUMN logical_clock_value BIGINT NOT NULL DEFAULT 0;
+
+UPDATE pg_catalog.pg_dist_local_group SET logical_clock_value = extract(epoch from now()) * 1000;
+
+DROP FUNCTION IF EXISTS pg_catalog.get_cluster_clock();
+
+CREATE OR REPLACE FUNCTION pg_catalog.get_cluster_clock()
+  RETURNS BIGINT
+  LANGUAGE C STABLE PARALLEL SAFE STRICT
+  AS 'MODULE_PATHNAME', $$get_cluster_clock$$;
+COMMENT ON FUNCTION pg_catalog.get_cluster_clock()
+     IS 'returns monotonically increasing logical clock value as close to epoch value (in milli seconds) possible, with the guarantee that it will never go back from its current value even after restart and crashes';
+
+
+CREATE OR REPLACE FUNCTION pg_catalog.set_transaction_id_clock_value(bigint)
```

Comment:
1) Not sure if pg_catalog is the correct schema for this UDF given that it's only supposed to be called by Citus internally, maybe should have this in `citus_internal` schema ?
2) Each function should have its own directory and x.y-z.sql & latest.sql files, so we should have a new directory for this UDF
3) Instead of XXXXX, let's have a better comment

Reply:
Thanks for the catch, XXXX was a place-holder to add the comment, and I forgot to change it.